pkg/options/server.go

```diff
@@ -38,6 +39,13 @@ func GetDefaultRateLimiterOptions() http.RateLimiterOptions {
 	}
 }
 
+func GetDefaultStorageOptions() http.StorageOptions {
+	return http.StorageOptions{
+		MaximumFileInputsMemoryMB: GetDefaultServeOptionInt("SERVER_MAX_FILE_INPUTS_MEMORY_MB", 1),
+		MaximumFileInputsSizeMB:   GetDefaultServeOptionInt("SERVER_MAX_FILE_INPUTS_SIZE_MB", 10),
```

Comment:
Just something to consider when we roll this out but we should probs up the Solver disk space on the machine

Reply:
Yep 💯